src/components/views/auth/InteractiveAuthEntryComponents.js

```diff
@@ -626,7 +752,7 @@ export const FallbackAuthEntry = createReactClass({
         }
         return (
             <div>
-                <a ref={this._fallbackButton} onClick={this._onShowFallbackClick}>{ _t("Start authentication") }</a>
+                <a href="" ref={this._fallbackButton} onClick={this._onShowFallbackClick}>{ _t("Start authentication") }</a>
```

Comment:
If you're changing this it should probably be an AccessibleButton styled with `mx_linkButton`

Reply:
Can we push that out to a different PR? We're missing designs, and the thing doesn't even work (either Riot is listening for the wrong thing, or Synapse isn't sending it). The goal here was just to get it it usable enough to do some light testing. I can also back out the change if preferred.

Reply:
OK, not critical, but maybe useful for future reference on the way to do these things.